### DISCUSSION TITLE
A MobileNetV2 Pre-trained model ?

### DISCUSSION THREAD

**Discussion by xiaofeinot** [STVIR/pysot#452] [Q&A]
Is there a MobileNetV2 Pre-trained model ? Cause I can only find the alexnet and resnet pre-trained model in the google link, but there's no MobileNetV2's. If any one knows how to train siam with pre-trained MobileNetV2 as backbone, please help me. Thanks a lot!

**Comment by ZhiyuanChen** [comment]
> Is there a MobileNetV2 Pre-trained model ?

I'm afraid no.
